flottform/demo/src/routes/custom-ui/+page.svelte

```diff
@@ -0,0 +1,586 @@
+			"I'm facing an unusual challenge where the system is asking me to upload a photo of myself wearing a unicorn costume while holding a rubber chicken. It's as if the authentication process has taken a whimsical turn into a surreal circus!",
+			'My transaction is on hold until I upload a selfie balancing a pineapple on my head while reciting the alphabet backward. Is this some sort of cognitive test or an initiation into the secret society of online shoppers?',
+			"To access the restricted area, the system demands a photo of me impersonating a famous historical figure of my choice while juggling three rubber ducks. It's like stepping into a time-traveling talent show!",
+			"I'm stuck in authentication limbo where the system insists on a photo of me wearing a pirate hat and holding a sign that says 'Arrr! I solemnly swear I'm up to no good'. Is this a security measure or a comedic treasure hunt?",
+			"I'm trying to unlock a premium feature, but the system demands a selfie of me wearing a Viking helmet while riding a hobby horse and reciting Shakespearean sonnets. To be or not to be a verified user, that is the question!",
+			"I'm experiencing a hardware glitch where my mouse pointer seems to be stuck in an infinite loop, circling endlessly. It's as if my computer is caught in a dance routine it can't escape. This quirky performance has turned my hardware into a dance floor, and now my head is spinning too—literally! I fear it might be broken. Is there a reboot button for humans as well?"
+		]
+	};
+
+	let isFillingOut = false;
+
+	async function typeValue(valueToFill: string, refValue: string, delay = 10) {
+		const letters = valueToFill.split('');
+		let i = 0;
+		$prefilledForm[refValue] = '';
+		while (i < letters.length) {
+			await waitForMs(delay);
+			$prefilledForm[refValue] += letters[i];
+			i++;
+		}
+		return;
+	}
+
+	function waitForMs(ms: number) {
+		return new Promise((resolve) => setTimeout(resolve, ms));
+	}
+
+	const fillOutForm = async () => {
+		resetValues();
+		isFillingOut = true;
+		await typeValue(randomArrayElement(mockUser.name), 'name');
+		await waitForMs(100);
+		await typeValue(randomArrayElement(mockUser.surname), 'surname');
+		await waitForMs(50);
+		await typeValue(
+			`${$prefilledForm.name}.${$prefilledForm.surname}@example.com`.toLocaleLowerCase(),
+			'email'
+		);
+		await waitForMs(80);
+		await typeValue(randomArrayElement(mockUser.phone), 'phone');
+		await waitForMs(110);
+		await typeValue(randomArrayElement(mockUser.street), 'street');
+		await waitForMs(40);
+		await typeValue(randomArrayElement(mockUser.houseNumber), 'houseNumber');
+		await waitForMs(80);
+		await typeValue(randomArrayElement(mockUser.city), 'city');
+		await waitForMs(50);
+		await typeValue(randomArrayElement(mockUser.postcode), 'postcode');
+		await waitForMs(70);
+		await typeValue(randomArrayElement(mockUser.problemDescription), 'problemDescription', 5);
+		highlighted = true;
+		isFillingOut = false;
+	};
+
```

Comment:
We can use `bind` to get access to the elements we created in out html template

```suggestion
	let createChannelButton: HTMLButtonElement;
	let dialog: HTMLDialogElement;
	let isOpenModal: boolean = false;
	let webRtcLink: string;
	let webRtcQrCode: string;
```